### PR TITLE
Use bundler

### PR DESCRIPTION
**Pull request by jherdman**
Update to use Bundler, and its various tools.

### CODE REVIEW

File: em-jack.gemspec [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  s.homepage    = 'https://github.com/dj2/em-jack/'
+  s.summary     = 'An evented Beanstalk client'
+  s.description = 'An evented Beanstalk client'
 
-  s.summary = %q{An evented Beanstalk client.}
-  s.description = %q{An evented Beanstalk client.}
+  s.required_rubygems_version = '>= 1.3.6'
 
-  s.add_dependency('eventmachine')
+  s.add_dependency 'eventmachine', ['>= 1.0.0.beta.3']
```
Comment (dj2) [inline em-jack.gemspec]: Jack should work with older versions of eventamchine. At a max, I think this should be 0.12.10 until 1.0 is out.

**Comment by dj2** [comment]
This is looking good. One thing, I'm not a fan of having .rvmrc files committed.  If you can remove that and bump the required EM version down this looks merge-able.

**Comment by jherdman** [comment]
Take a look at 8cadfff9733ee9b7f701846f034e8644c646848d. I think that should satisfy some projects I've encountered using the EventMachine betas, and those using older EventMachine.

**Comment by dj2** [comment]
Looks good, thanks for the fixes.

**Pull request merged**